tools/workflow/clean_aws_resources.go

```diff
@@ -33,18 +33,25 @@ const (
 )
 
 func main() {
-	log.Printf("Beging terminating EC2 Instances")
-	terminateEc2Instances()
+	awsRegions := []string{"us-west-2","us-east-2"}
```

Comment:
should we define these regions outside of the main func with the other const?

Reply:
True though. I intend to passed these regions as environment variables but nice idea if I did not manage to do that.